docs/cluster-management/configure-api.md

```diff
@@ -242,42 +246,51 @@ Set these environment variables:
 
 | Environment name           | Required                  | Default Value | Description                                  |
 |:---------------------------|:--------------------------|:--------------|:---------------------------------------------|
-| SCM_PLUGIN                 | No                        | github        | `github` or `bitbucket`                      |
+| SCM_SETTINGS               | Yes                       |               | JSON object with SCM settings                |
 | SECRET_OAUTH_CLIENT_ID     | Yes                       |               | Your OAuth Client Id (Application key)       |
 | SECRET_OAUTH_CLIENT_SECRET | Yes                       |               | You OAuth Client secret (Application secret) |
 | WEBHOOK_GITHUB_SECRET      | Yes for Github            |               | Secret to sign for webhooks                  |
 | SCM_GITHUB_GHE_HOST        | Yes for Github Enterprise |               | GHE host for Github Enterprise               |
-| SCM_PRIVATE_REPO_SUPPORT   | No                        | false         | Ask Github users for 'repo' scope to allow read/write access to public and private repo |
-| SCM_USERNAME               | No                        | sd-buildbot   | Username for checkout                        |
-| SCM_EMAIL                  | No                        | dev-null@screwdriver.cd | Email of user for checkout         |
+| SCM_PRIVATE_REPO_SUPPORT   | Yes for Github            | false         | Ask Github users for 'repo' scope to allow read/write access to public and private repo |
+| SCM_USERNAME               | Yes for Github            | sd-buildbot   | Username for checkout                        |
+| SCM_EMAIL                  | Yes for Github            | dev-null@screwdriver.cd | Email of user for checkout         |
 
 
 ##### Github:
 ```yaml
 # config/local.yaml
-scm:
-    plugin: github
-    github:
-        oauthClientId: YOUR-OAUTH-CLIENT-ID
-        oauthClientSecret: YOUR-OAUTH-CLIENT-SECRET
-        # Secret to add to GitHub webhooks so that we can validate them
-        secret: SUPER-SECRET-SIGNING-THING
-        # You can also configure for use with GitHub enterprise
-        # gheHost: github.screwdriver.cd
-        # Whether to support private repo
-        # privateRepo: true
+scms: {
```

Comment:
For this, I don't think you can put json directly in local.yaml..
If they use environment variable for scmsettings, then that env `SCM_SETTINGS` can be in json